frontend/ui/quickstart.lua

```diff
@@ -228,6 +228,36 @@ function QuickStart:getQuickStart()
 
         local quickstart_html = FileConverter:mdToHtml(quickstart_guide, _("KOReader Quickstart Guide"), stylesheet)
         if quickstart_html then
+            -- Fix links to images, which are in KOReader install directory, which may not
+            -- be alongside help/ on some platforms like Android.
+            -- crengine won't accept full paths, so we need to make these relative
+            local src = FFIUtil.realpath(quickstart_dir) .. "/"
+            local dst = lfs.currentdir() .. "/"
+            -- Find the common leading directories
+            local idx = 0
+            while true do
+                local tst = src:find("/", idx + 1, true)
+                if tst and src:sub(1,tst) == dst:sub(1,tst) then
+                    idx = tst
+                else
+                    break
+                end
+            end
+            -- Trim off the common directories from the front
+            src = src:sub(idx + 1)
+            dst = dst:sub(idx + 1)
+            -- Back up from dst to get to this common parent
+            local relpath = ""
+            idx = src:find("/")
+            while idx do
+                relpath = relpath .. "../"
+                idx = src:find("/", idx + 1)
+            end
+            -- Add the path down to dst from here
+            relpath = relpath .. dst
+            relpath = relpath:gsub("//", "/") -- make it prettier
+            quickstart_html = quickstart_html:gsub([[src="resources/]], [[src="]]..relpath..[[resources/]])
+            -- Write the fixed HTML content
```

Comment:
Would this make more sense in DataStorage?

It's effectively already used as a fallback, but I mean more as something like runtimeDir vs dataDir.
https://github.com/koreader/koreader/blob/2222f25422487e726db6c793de85af356e5fd493/datastorage.lua#L42-L52

Reply:
I don't think so, it somehow happens after we have used DataStorage: we have some path that happens to be in a subdir of DataDir, and we want from it a relative path to the current/runtime directory (if it was the other way around, then, may be).

I thought about putting it into util.lua (`util.getRelativePathFromTo(from, to)`) in case we need it elsewhere, but there's a few options/decisions in this code that are fine for this ues case but may be not others, and I don't want to handle and make parameters :)